package.json

```diff
@@ -17,8 +17,11 @@
     "@babel/preset-env": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
     "@babel/runtime": "^7.0.0",
-    "@sentry/browser": "5.2.0-beta.0",
-    "@sentry/integrations": "5.2.0-beta.0",
+    "@sentry/browser": "^5.4.2",
+    "@sentry/integrations": "^5.4.2",
+    "@sentry/typescript": "^5.3.0",
+    "@types/lodash": "^4.14.134",
```

Comment:
Should these be dev deps?

Reply:
I think these are needed for the production build right now, but we should probably look into that. @HazAT - I know you mentioned something about this earlier, is that accurate?